src/main/python/pre_fx_data.py

```diff
@@ -206,6 +210,47 @@ def load_data_set_from_nwb(self, path: str):
                 err
             )
 
+    def extract_manual_sweep_states(self):
+        """ Extract manual sweep states in the format schemas.ManualSweepStates
+        from PreFxData
+        """
+
+        return [
```

Comment:
?

Reply:
don't you want the actual sweep states?

Reply:
I do. I did not implement it because we did not yet have an ability to pass manual states to the the PreFxData. But, if I define the schema for the PreFxdata then I can write the function. I just did not realize that this was a part of this issue.

Reply:
@NileGraddis will provide  functionality for extracting sweeps states